src/execution/deprecated.c

```diff
@@ -0,0 +1,154 @@
+#include "deprecated.h"
+#include "modelRun_ctx.h"
+#include "util/string_utils.h"
+
+static int _parseTimeout(RedisModuleString *timeout_arg, RAI_Error *error, long long *timeout) {
```

Comment:
we have, and will use this function in multiple places
move this to a shared location